.travis.yml

```diff
@@ -1,7 +1,4 @@
-language: csharp
-dotnet: 2.1.301
-mono: none
-dist: trusty
+language: minimal
```

Comment:
I changed it back to csharp to let our unit tests run. Pretty sure this change will give build issues.

Reply:
Put original definitions back in there